### DISCUSSION TITLE
FDS installing for mac

### DISCUSSION THREAD

**Discussion by NaokiYamamoto-KCFB** [firemodels/fds#13896] [Q&A]
MacBookPro:~ naokiyamamoto$ fds simple_test.fds
[MacBookPro:20412] [[INVALID],INVALID] ORTE_ERROR_LOG: A system-required executable either could not be found or was not executable by this user in file ess_singleton_module.c at line 572
[MacBookPro:20412] [[INVALID],INVALID] ORTE_ERROR_LOG: A system-required executable either could not be found or was not executable by this user in file ess_singleton_module.c at line 172
--------------------------------------------------------------------------
Sorry!  You were supposed to get help about:
    orte_init:startup:internal-failure
But I couldn't open the help file:
    /opt/openmpi415_oneapi22u3/share/openmpi/help-orte-runtime: No such file or directory.  Sorry!
--------------------------------------------------------------------------
--------------------------------------------------------------------------
Sorry!  You were supposed to get help about:
    mpi_init:startup:internal-failure
But I couldn't open the help file:
    /opt/openmpi415_oneapi22u3/share/openmpi/help-mpi-runtime.txt: No such file or directory.  Sorry!
--------------------------------------------------------------------------
*** An error occurred in MPI_Init_thread
*** on a NULL communicator
*** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
***    and potentially your MPI job)
[MacBookPro:20412] Local abort before MPI_INIT completed completed successfully, but am not able to aggregate error messages, and not able to guarantee that all other processes were killed!

The following error message appears.
I have tried to re-download OpenMPI, etc.

**Comment by rmcdermo** [comment]
Have you followed the instructions [here](https://github.com/firemodels/fds/wiki/Installing-and-Running-FDS-under-macOS)?
